app/scripts/components/leaflet/waterpoint-marker.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { Marker, Popup } from 'react-leaflet';
+import { Map } from 'leaflet';
+
+const WaterpointMarker = React.createClass({
+  propTypes: {
+    LATITUDE: PropTypes.number.isRequired,
+    LONGITUDE: PropTypes.number.isRequired,
+    WATER_POINT_CODE: PropTypes.string.isRequired,
+    WATER_POINT_NAME: PropTypes.string.isRequired,
+    children: PropTypes.node,
+    map: PropTypes.instanceOf(Map),  // injectd by <Map>
+  },
+  render() {
+    const { children, map, LATITUDE, LONGITUDE } = this.props;
+    const clonedChildren = React.Children.map(children, child =>
```

Comment:
In this case we don't, so I took it out.

If we had children that could be react-leaflet components, then we would need to clone them to ensure they have the `map` prop.